src/component/room/_room.scss

```diff
@@ -1,15 +1,60 @@
-.close-save {
+@import '_vars';
+
+// * {
+//   border: solid red .25px;
+// }
+
+.info-button-container {
+  display: grid;
+  grid-template-columns: repeat(4, 25%);
+  grid-template-rows: repeat(4, 25%);
+}
+
+.active-voters {
+  grid-row: 1 / span 1; 
```

Comment:
what is `span`? How does that work, is that supposed to be a var?

Reply:
span tells it how many rows to go to after starting at row 1. There are many ways to do it, especially when it only goes one gridline, but i wanted to keep them a consistent format of " gridlinestartt# / span gridlinestospan#"